backend/services/reaction.py

```
@@ -0,0 +1,77 @@
+from zoneinfo import ZoneInfo
+from crud.notification import create_notification
+
+# Helper: notify user if not the actor
+
+def notify_if_not_self(db, actor_id, recipient_id, notif_type, post_id):
+    if actor_id != recipient_id:
+        create_notification(db, recipient_id, actor_id, notif_type, post_id)
+
+# Helper: like handling
+
+def update_like_count(like_data, db: Session, action: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `update_like_count`

Add return type annotation: `None`

(ANN201)